test/WebSites/TagHelpersWebSite/Controllers/Catalog_CacheTagHelperController.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace TagHelpersWebSite.Controllers
+{
+    public class Catalog_CacheTagHelperController : Controller
+    {
+        [HttpGet("/catalog")]
+        public ViewResult Splash(int categoryId, int correlationId)
+        {
+            var category = categoryId == 1 ? "Laptops" : "Phones";
+            ViewData["Category"] = category;
+            ViewData["Locale"] = Request.Headers["Locale"] == "N" ? "North" : "Default";
```

Comment:
nit: why the indirection?  that is why not have the client send "North" or use "N" here?